test/unit_tests/molecules/Multiselect-spec.js

```diff
@@ -71,7 +73,7 @@ describe( 'Multiselect', function() {
       option.value = 'Foo\'';
       multiselect.init();
       selectDom = document.querySelectorAll( 'select[multiple]' );
-      multiselectDom = document.querySelectorAll( '.cf-multi-select' );
+      multiselectDom = document.querySelectorAll( 'div.cf-multi-select' );
```

Comment:
Related to the question above, if the classname isn't used to the native select, this won't need the `div` chaining.